storage/innobase/include/api0api.h

```diff
@@ -651,22 +651,26 @@ ib_open_table_by_name(
 Insert a row to a table.
 @return	DB_SUCCESS or err code */
 
+#pragma namemanglingrule(fnparmtype, on)
```

Comment:
What does it fix?  Does source break some rule to which AIX compiler is sensitive? Can it be fixed without any platform-specific stuff